website/src/util/defaultDateRangeOption.ts

```diff
@@ -0,0 +1,44 @@
+import { type DateRangeOption, dateRangeOptionPresets } from '@genspectrum/dashboard-components/util';
+
+export const defaultDateRangeOption = {
+    since2000: { label: 'Since 2000', dateFrom: '2000-01-01' },
+    since2020: { label: 'Since 2020', dateFrom: '2020-01-01' },
```

Comment:
We don't need it twice ;)
```suggestion
    since2000: { label: 'Since 2000', dateFrom: '2000-01-01' },
```

Reply:
One of it is 2020 the other 2000.